bin/create-scripts/block.js

```diff
@@ -1,9 +1,11 @@
 /** @format */
 const path = require( 'path' );
 const webpack = require( 'webpack' );
+const fs = require( 'fs' );
+const sass = require( 'node-sass' );
 
 const __rootDir = path.resolve( __dirname, '../../' );
-const entryPath = path.resolve( process.argv[ 2 ] );
+const entryPath = path.resolve( path.join( process.argv[ 2 ], 'index.js' ) );
```

Comment:
In a next iteration, we might want to take entry points as defined in a `package.json` into account (i.e. the `main` field).